### DISCUSSION TITLE
Implementation of the ControlNetPlus  in the Fooocus Project

### DISCUSSION THREAD

**Discussion by bobsponge76** [lllyasviel/Fooocus#3264] [General]
I would like to ask if the implementation of the  new ControlNetPlus functionality  [https://github.com/xinsir6/ControlNetPlus/tree/main] for SDXL in the Fooocus project would improve even more control over our creations. Thank you.

**Comment by mashb1t** [comment]
Would have to evaluate the quality of ControlNetPlus first before making a final statement.

**Reply by dendenmushy** [reply]
Thanks 🙏
